test/e2e/common/downward_api.go

```diff
@@ -104,6 +104,61 @@ var _ = ginkgo.Describe("[sig-node] Downward API", func() {
 		testDownwardAPI(f, podName, env, expectations)
 	})
 
+	/*
+	   Release : v1.16
+	   Testname: DownwardAPI, environment for host ip and pod ip with host network
+	   Description: Downward API MUST expose Pod and Container fields as environment variables. Specify host IP and pod IP as environment variable in the Pod Spec are visible at runtime in the container if the pod uses host network. The host IP and the pod IP has to be the same
+	*/
+	framework.ConformanceIt("should provide host IP and pod IP as an env var if pod uses host network [NodeConformance]", func() {
```

Comment:
I'm pretty sure tests are only supposed to become Conformance tests after introduction and then further review.

Reply:
I was looking for some guidance, will try to ask tomorrow

Reply:
asked in #k8s-conformance on slack